sources/qa-lang-admin.php

```diff
@@ -2,134 +2,134 @@
-       'mailing_title'=>'Mailing', 
-       'mailing_unsubscribe'=>'Odjavi povezava bo dodana na dnu vsakega sporočila.', 
-       'maintenance_admin_only'=>'Vaša spletna stran je v ^ 1Vzdrževanje ^ 2 in je trenutno nedostopna rednih uporabnikov.', //Your site is in ^1maintenance^2 and is currently inaccessible to regular users.
+       'mailing_progress'=>'Poslano ^ 1 od ^ 2 uporabnikom ...', //Mailing completed for ^1 of ^2 users...
+       'mailing_title'=>'Naslov (zadeva)', 
+       'mailing_unsubscribe'=>'Odjavi se od prejemanja E-pošte.', 
+       'maintenance_admin_only'=>'Vaša spletna stran je v ^ 1Vzdrževanje ^ 2 in je trenutno nedostopna rednim uporabnikom.', //Your site is in ^1maintenance^2 and is currently inaccessible to regular users.
        'maximum_x'=>'(Max ^)', // (max ^)
-       'moderate_title'=>'Zmerna', 
-       'module_x_database_init'=>'^ 1 ^ 2 modula zahteva nekaj ^ 3database inicializacija ^ 4.', //The ^1 ^2 module requires some ^3database initialization^4.
-       'most_flagged_title'=>'Označena vsebina', 
+       'moderate_title'=>'Uredi naslov', 
+       'module_x_database_init'=>'^ 1 ^ 2 modul zahteva ^ 3database inicializacija ^ 4.', //The ^1 ^2 module requires some ^3database initialization^4.
+       'most_flagged_title'=>'Naslov z največ zastavicami/opozorili', 
        'mysql_version'=>'MySQL verzija:', 
        'nav_links_explanation'=>'Prikaži navigacijske povezave:', 
        'nav_qa_is_home'=>'Q & A (povezave na spletno stran)', 
-       'neat_urls_note'=>'(Zahteva ^ 1htaccess ^ 2 datoteke)', // (requires ^1htaccess^2 file)
-       'no_approve_found'=>'Ni vsebine čaka na odobritev', 
-       'no_classification'=>'Noben', 
-       'no_flagged_found'=>'Ne označeni ugotovljena vsebnost', 
-       'no_hidden_found'=>'Ne skrito vsebino najti', 
+       'neat_urls_note'=>'(Zahteva ^ 1htaccess ^ 2 datoteko)', // (requires ^1htaccess^2 file)
+       'no_approve_found'=>'Ni odobritve', 
+       'no_classification'=>'Ni klasifikacije', 
+       'no_flagged_found'=>'Ni najdenih zastavic', 
+       'no_hidden_found'=>'Ni skritih vsebin', 
        'no_image_gd'=>'Nameščena različica PHP je bila prevedena brez podpore GD slike, tako da uporabniki ne morejo neposredno naložiti svoih avatarjev.', 
        'no_link'=>'Ni povezave', 
        'no_multibyte'=>'Nameščena različica PHP je bil prevedena brez podpore večbajtnega niza. Iskanje bo manj učinkovito za ne-latinico.', 
-       'no_privileges'=>'Samo administratorji lahko dostopajo do te strani.', 
+       'no_privileges'=>'Nimaš dostopa do te strani.', 
        'no_unapproved_found'=>'Ni uporabnikov, ki čakajo na odobritev', 
-       'not_logged_in'=>'Prosimo, ^1log in ^2 kot admin za dostop do te strani.', //Please ^1log in^2 as the administrator to access this page.
+       'not_logged_in'=>'Prosimo, ^1log in ^2 kot administrator za dostop do te strani.', //Please ^1log in^2 as the administrator to access this page.
```

Comment:
Prosim bodite pozorni na primere kot je ta. Če sta v angleškem stringu in^2 skupaj, morata biti skupaj in neprevedena tudi v slo stringu. Podobno zgoraj pri ^1htaccess^2.